modules/peers.js

```diff
@@ -122,17 +240,16 @@ __private.updatePeersList = function (cb) {
 					library.logger.error(['Rejecting peer', peer.string, 'with incompatible version', peer.version].join(' '));
 					self.remove(peer.ip, peer.port);
 				} else {
-					delete peer.broadhash;
-					delete peer.height;
-					self.update(peer);
+					// We currently don't base on those data, but only discover new peers here - so insert-only
```

Comment:
`We currently don't base on those data, but only discover new peers here - so insert-only` > `We only discover new peers here - so insert-only`

Not sure what you mean by: `We currently don't base on those data`.